.github/workflows/build.yaml

```
@@ -1,26 +1,363 @@
+
+        - name: Install
+          run: install -D -t release/${{ matrix.target }} ${{ matrix.install_src_glob }}
+
+        - name: Upload
+          uses: actions/upload-artifact@v3
+          with:
+            name: release_${{ matrix.target }}
+            path: release/${{ matrix.target }}
+
+    macos_10_15:
+        name: MacOS 10.15 Build
         runs-on: macos-10.15
 
         steps:
-            - name: "Clone wake"
+            - name: Clone wake
               uses: actions/checkout@v2
               with:
                 fetch-depth: 0
 
-            - name: "Install deps"
+            - name: Install deps
               run: |
                 brew install --cask osxfuse
                 brew install re2 || brew install re2 --head
 
-            - name: "Build wake"
+            - name: Build wake
               run: make all -j3
 
-            - name: "Test wake"
+            - name: Test wake
               run: make test
+
+    docs:
+      runs-on: ubuntu-latest
+      needs: tarball
+      name: Docs
+      steps:
+        - name: Download index.html
+          uses: actions/download-artifact@v3
+          with:
+            name: index.html
+
+        - name: Download html
+          uses: actions/download-artifact@v3
+          with:
+            name: html
+
+        - name: Make .nojekyll
+          run: touch .nojekyll
+
+        - name: Upload
+          uses: actions/upload-pages-artifact@v1
+          with:
+            path: .
+
+    deploy_docs:
+      # Only run the 'deploy_docs' job if this workflow was triggered by a push to master
+      if: github.event_name == 'push' && github.ref_type == 'branch' && github.ref_name == 'master'
+
+      permissions:
+        pages: write
+        id-token: write
+
+      environment:
+        name: github-pages
+        url: ${{ steps.deployment.outputs.page_url }}
+
+      runs-on: ubuntu-latest
+      needs: docs
+      name: Deploy Docs to GH Pages
+      steps:
+        - name: Deploy
+          id: deployment
+          uses: actions/deploy-pages@v1
+
+    release:
+      # Only run the 'release' job if this workflow was triggered by the creation of a tag
+      if: github.event_name == 'push' && github.ref_type == 'tag'
+
+      runs-on: ubuntu-latest
+      needs: [tarball, docker_builds]
+      name: Create Release
+      steps:
+        - name: Download Tarball
+          uses: actions/download-artifact@v3
+          with:
+            name: tarball
+            path: tarball
+
+        - name: Download VSCode
+          uses: actions/download-artifact@v3
+          with:
+            name: release_vscode
+            path: vscode
+
+        - name: Download Alpine
+          uses: actions/download-artifact@v3
+          with:
+            name: release_alpine
+            path: apline
```

I don't see any typos beyond this one.
```suggestion
            path: alpine
```